src/main.cpp

```diff
@@ -27,14 +27,58 @@
 #include <modulemanager.h>
 #include <licensesystem.h>
 #include <customerdatasystem.h>
+#include <QCommandLineParser>
+
+
+QString getDevNameFromUBoot()
+{
+    QString strDeviceName;
+    // Check for kernel cmdline param which u-boot should set
+    QFile procFileCmdLine(QLatin1String("/proc/cmdline"));
+    if(procFileCmdLine.open(QIODevice::ReadOnly))
+    {
+        QString cmdLine = procFileCmdLine.readAll();
+        procFileCmdLine.close();
+        // Extract 'zera_device=<device_name>'
+        QRegExp regExp(QLatin1String("\\bzera_device=[^ ]*"));
+        if(regExp.indexIn(cmdLine) != -1)
+        {
+            strDeviceName = regExp.cap(0);
+            // The following should go in regex above but...
+            strDeviceName.replace(QLatin1String("zera_device="), QLatin1String(""));
+            strDeviceName.replace(QLatin1String("\n"), QLatin1String(""));
+            qInfo() << "ZERA Device from kernel cmdline: " << strDeviceName;
+        }
+    }
+    return strDeviceName;
+}
+
 
```

Comment:
Where has this been removed? It was there ever since. Please check because it makes searching for changes more complicated than necessary.